README.md

```diff
@@ -12,17 +12,17 @@ Start to log everything in just few clicks
 ## Installing
 
 ```bash
-$ npm install loggie-io
+$ npm install crashdeck-io
```

Comment:
I think we want the name of the package to just be `crashdeck` rather than `crashdeck-io`